contracts/contracts/strategies/ConvexEthMetaStrategy.sol

```diff
@@ -198,9 +275,13 @@ contract ConvexEthMetaStrategy is InitializableAbstractStrategy {
         // slither-disable-next-line unused-return
         curvePool.remove_liquidity(requiredLpTokens, _minWithdrawalAmounts);
 
-        // Burn OETH
-        IVault(vaultAddress).burnForStrategy(oeth.balanceOf(address(this)));
-        // Transfer WETH
+        // Burn all the removed OETH and any that was left in the strategy
+        uint256 oethToBurn = oeth.balanceOf(address(this));
+        IVault(vaultAddress).burnForStrategy(oethToBurn);
+
+        emit Withdrawal(address(oeth), address(lpToken), oethToBurn);
+
+        // Transfer WETH to the recipient
         weth.deposit{ value: _amount }();
         require(
```

Comment:
this is implementing a safe transfer. Should we use the following instead?
```Solidity
weth.safeTransfer(_recipient, _amount);
```

Reply:
We don't need the safe behavior on weth, it's well behaved. Only have to use safeTransfer when we are dealing with coins with incompatible interfaces like USDT. If we've specified a specific coin, and we know it's good, I'm good with just the transfer.

Reply:
ok, thanks